Gemfile

```diff
@@ -0,0 +1,37 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+
+gem 'rails', '~> 5.1.5'
+
+gem 'pg', '>= 0.18', '< 2.0'
+
+gem 'puma', '~> 3.7'
+
+gem 'sass-rails', '~> 5.0'
+
+gem 'uglifier', '>= 1.3.0'
+
+gem 'coffee-rails', '~> 4.2'
+
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
+
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
Please review what debugger you use. Pry is more in line with the standard we use at Craft